controllers/comment-controller.js

```diff
@@ -0,0 +1,19 @@
+const { removeCommentId } = require("../models/comment-models");
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  if (!parseInt(comment_id)) {
+    res.status(400).send({ msg: `${comment_id} is invalid` });
+  }
+
+  removeCommentId(comment_id)
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return res
+          .status(404)
+          .send({ msg: `No Comment found under ${comment_id}` });
+      }
+      res.status(204).send({});
+    })
```

Comment:
There are a few things I can pick up on from this controller that could be considered for changes:
- You are checking whether the `comment_id` is a number using `parseInt`, this is not necessary as PSQL will throw an error with a code of '22P02' if the data type is not valid when attempting to run the query
- You are checking the `rows.length` to see if the _comment_id_ existed, this is fine but I would recommend moving this logic into your model as the model is responsible for data validation and throwing errors 👍